corehq/apps/data_cleaning/forms/cleaning.py

```diff
@@ -212,3 +213,26 @@ def clean(self):
                 return cleaned_data
 
         return cleaned_data
+
+    def create_bulk_edit_change(self):
+        prop_id = self.cleaned_data["clean_prop_id"]
+        action_type = self.cleaned_data["clean_action"]
+        change_kwargs = {
+            EditActionType.REPLACE: {
+                "replace_string": self.cleaned_data["replace_all_string"],
+            },
+            EditActionType.FIND_REPLACE: {
+                "find_string": self.cleaned_data["find_string"],
+                "replace_string": self.cleaned_data["replace_string"],
+                "use_regex": self.cleaned_data["use_regex"],
+            },
+            EditActionType.COPY_REPLACE: {
+                "copy_from_prop_id": self.cleaned_data["copy_from_prop_id"],
+            },
+        }.get(action_type, {})
+        return BulkEditChange.objects.create(
+            session=self.session,
+            prop_id=prop_id,
+            action_type=action_type,
+            **change_kwargs,
+        )
```

Comment:
Currently this can create empty change objects for the `action_type`s not handled here. I assume this is just a temporary thing, and/or an empty change doesn't really pose any issues?

Reply:
there are action types that don't require additional kwargs, like `TITLE_CASE` etc.